setup.py

```diff
@@ -47,8 +47,9 @@ def main():
         install_requires=reqs,
         extras_require={
             'test': test_reqs,
-            'ipy3': ['ipython[test,notebook]<4.0'],
-            'ipy4': ['ipython<6.0', 'notebook[test]>=4.0,<5.0'],
+            'ipy3': ['ipython[test,notebook]<4.0', 'tornado<5.0'],
+            'ipy4': ['ipython<6.0', 'notebook[test]>=4.0,<5.0', 'tornado<5.0'],
```

Comment:
Why the tornado change? I wouldn't expect pgcontents to care much about the version of tornado.

Reply:
pgcontents doesn't care, but notebook very much does and older versions of notebook don't pin tornado to an upper bound.  tornado 5 did enough backward incompatible changes to basically make it not work unless you are running a substantially closer to master version of notebook.